### ISSUE TITLE
Fix CMS data migrations or replace them with management command(s)

### ISSUE THREAD

**Issue by ahmed-belal** [mitodl/bootcamp-ecommerce#900]
Modify CMS setup migrations to be no-ops and create equivalent management commands.

#### Background (from Slack):
The issue is, in order for the CMS setup to be in a migration (so people didn't have to manually run a command) we had to directly import the actual model from `cms.models`. That broke Django and Wagtail convention. The issue here is because only dummy/basic models are available at migration time we broke the convention mentioned above so we could do Wagtail tree operations, hoping the main `wagtail.core.models.Page` model would not change. In `2.7.2` the model has changed so our migrations break upon application. We'd need to empty out those migrations to be no-ops and write equivalent management commands so that the upgrade can go through.

**Comment by gsidebo** [comment]
@ahmed-belal I remember we imported `cms.models` at the top level just b/c the normal import wouldn't work in the body of the migration. I saw [this article](https://www.caktusgroup.com/blog/2019/09/12/wagtail-data-migrations/) and it shows an importing technique that we should try:

```python
    try:
        HomePage = import_module('cms.models').HomePage
    except:
        # If HomePage model no longer exists, return directly
        return
```

**Comment by ahmed-belal** [comment]
@gsidebo the problem isn't the way it's being imported. Problem is, wagtail `2.7.2` includes a migration that adds a field on the `Page` model. Now that we upgrade the dependency, any and all data migrations that have to do with the `Page` model do not go through because the underlying code has changed in the core wagtail app. It tries to access the new field (IIRC `Page.disabled_at`) and the migration fails because the field does not exist yet.

Now we can't update the existing migrations and add the new migration as a pre-req to it because systems that already have our migrations applied will start throwing an inconsistent migration history errors. We'll have to empty out those migrations into no-ops and create counterpart idempotent management commands.

**Comment by gsidebo** [comment]
@ahmed-belal Ouch... Could we make the initial CMS migration dependent on that new migration? These are the current dependencies for our first CMS migration (`cms/migrations/0001_initial.py`):

```python
    dependencies = [
        ("wagtailimages", "0001_squashed_0021"),
        ("wagtailcore", "0045_assign_unlock_grouppagepermission"),
    ]
```

If we pinpoint the correct migration in Wagtail, I think we could change the dependency in this initial migration or one of our other CMS migrations. 

Also, I'm not sure if this matters, but I took a look at their [`2.7.2` tag](https://github.com/wagtail/wagtail/commits/v2.7.2) and it doesn't seem like it adds any migrations. Just want to flag that in case we're actually upgrading beyond the version that we're expecting.

**Comment by gsidebo** [comment]
@ahmed-belal One last thing we can try: just wrap all of the `Page` imports in a try/except. Those migrations are just meant to put sensible defaults in place so the app works when you first run it, so it's fine to just continue as normal if the import fails. If the import fails, we can log a warning that tells the user to fully migrate Wagtail before running the CMS migrations and try again

**Comment by ahmed-belal** [comment]
@gsidebo the import does not fail, it succeeds. So the situation is this:

1. We are able to import the latest "version" of the Page class (which includes doing stuff with the new attributes added to the model lately)
2. Because the latest migration for Page model has not run yet, we get a field does not exist error as soon as we try to create/publish/save any Page.

I think we don't need to worry about this anymore, I moved stuff out of the CMS configure data migration (the one that setup the homepage) and updated Wagtail in requirements, bumped some linter/tester version numbers, tiny fix in `models.py` (actual model class F-keys, not string-based F-keys) and it seemed to work. I just need to convert the configure CMS management command (equivalent to the migration I emptied out) into being an idempotent process and we should be good to go.
